cirq-core/cirq/circuits/circuit.py

```diff
@@ -1153,32 +1153,32 @@ def to_text_diagram_drawer(
         qubits: Tuple[Any, ...] = ops.QubitOrder.as_qubit_order(qubit_order).order_for(
             self.all_qubits()
         )
-        cbits = tuple(
+        cbits: Tuple[Any, ...] = tuple(
```

Comment:
Is it possible for `qubits` and `cbits` to have types `Tuple[Qid, ...]` and `Tuple[MeasurementKey, ...]`, respectively? Similarly, can `labels` have type `Tuple[Union[Qid, MeasurementKey], ...]`?